.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
```

Comment:
nope, i dont see that in documentation, just Trusty (14) and Precise (12), and no news in latest changelog https://docs.travis-ci.com/user/build-environment-updates/2016-08-24/